website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -149,30 +181,34 @@ When a variable set is priority, the values take precedence over any variables w
 
 For example, variables in a priority global variable set would take precedence over all variables with the same key.
 
-If two priority variable sets with the same scope include the same variable key, HCP Terraform will determine precedence by the alphabetical order of the variable sets' names.
+If two priority variable sets with the same scope and ownership include the same variable key, HCP Terraform will determine precedence by the alphabetical order of the variable sets' names.
 
 While a priority variable set can enforce that Terraform variables use designated values, it does not guarantee that the configuration uses the variable. A user can still directly modify the Terraform configuration to remove usage of a variable and replace it with a hard-coded value. For stricter enforcement, we recommend using policy checks or run tasks.
 
 ## Precedence example
 
 Consider an example workspace that has the following variables applied:
 
-| (**Scope**) Source                         | Region      | Var1 | Replicas  |
-| ------------------------------------------ | ----------- | ---- | --------- |
-| Priority **global** variable set           | `us-east-1` |      |           |
-| Priority **project-scoped** variable set   | `us-east-2` |      |           |
-| Priority **workspace-scoped** variable set | `us-west-1` |      |           |
-| Command line argument                      | `us-west-2` |      |   `9`     |
-| Local environment variable                 |             |      |   `8`     |
-| **Workspace-specific** variable            |             | `h`  |   `1`     |
-| **Workspace-scoped** variable set          |             | `y`  |   `2`     |
-| **Project-scoped** variable set            |             |      |   `3`     |
-| **Global** variable set                    |             |      |   `4`     |
+| (**Scope**) Source                                            | Region      | Var1 | Replicas |
+| ------------------------------------------------------------- | ----------- | ---- | -------- |
+| Priority **global** variable set                              | `us-east-1` |      |          |
+| Priority **project-scoped** organization-owned variable set   | `us-east-2` |      |          |
```

Comment:
To make the differences between rows more apparent, how would you feel about moving the ownership changes to the left so they can more visible? Bringing the focus to the ownership instead of just the scope.

For example, moving organization-owned before "project-scoped":
```suggestion
| Priority organization-owned **project-scoped** variable set   | `us-east-2` |      |          |
```

Reply:
We could also have an ownership column to really drive the point home. What do you think?

Reply:
I tried out having extra columns for the different factors (priority, ownership, scope) but then I though it's kind of confusing to then also have the columns for the example variables. 🤔  It looks like this:

| Source                                                        | Priority | Ownership    | Scope     | Region      | Var1 | Replicas |
| ------------------------------------------------------------- | -------- | ------------ | --------- | ----------- | ---- | -------- |
| Priority **global** variable set                              | true     | organization | global    | `us-east-1` |      |          |
| Priority organization-owned **project-scoped** variable set   | true     | organization | project   | `us-east-2` |      |          |
| Priority project-owned **project-scoped** variable set        | true     | project      | project   | `eu-east-2` |      |          |
| Priority organization-owned **workspace-scoped** variable set | true     | organization | workspace | `us-west-1` |      |          |
| Priority project-owned **workspace-scoped** variable set      | true     | project      | workspace | `eu-west-1` |      |          |
| Command line argument                                         | N/A      |              | workspace | `us-west-2` |      | `9`      |
| Local environment variable                                    | N/A      |              | workspace |             |      | `8`      |
| **Workspace-specific** variable                               | N/A      |              | workspace |             | `h`  | `1`      |
| Project-owned **workspace-scoped** variable set               | false    | project      | workspace |             | `y`  | `2`      |
| Organization-owned **workspace-scoped** variable set          | false    | organization | workspace |             | `z`  | `3`      |
| Project-owned **project-scoped** variable set                 | false    | project      | project   |             |      | `4`      |
| Organization-owned **Project-scoped** variable set            | false    | organization | project   |             |      | `5`      |
| **Global** variable set                                       | false    |              | global    |             |      | `6`      |

Reply:
Oh geeze yeah, that's rough 😂  I think it makes sense to skip the ownership column then!

Reply:
Not sure if now it's too much information in this section, but I added a second, separate table with all the factors that are used to determine the precedence. I liked having the columns with the factors because I think it's easier to view the precedence order and how the factors affect it. Let me know what you think!